src/intrinsicTypes.ts

```diff
@@ -0,0 +1,31 @@
+import { type INodeWritableProps, type ITextNodeWritableProps } from "@lightningjs/renderer";
+
+export interface KeyMap {
```

Comment:
I think this is more generic... Any keycode to name type map (and these are just some defaults)